### PR TITLE
WorkinOnIt

### PR DESCRIPTION
**Pull request by mfmeier**

### CODE REVIEW

**Comment by sumeetjain** [comment]
Hi Michael. I saw your question via email. If possible, let's keep the conversation here on GitHub. It's a lot easier to talk about code here.

I know you'll probably see this message in your email inbox, but click https://github.com/omahacodeschool/ruby-toy__check-if-anagrams/pull/13 to see it on GitHub itself. This is the page for your assignment submission–also known as a "Pull Request".

I can much more quickly respond to questions here.

So my question for you, based on your email, is: Can you save and run the `git` steps so that your code shows up here? Or is your code already here in this Pull Request? Once I see your updated code, I can help you figure out why you're seeing that error.

**Comment by mfmeier** [comment]
Ok, I just did

File: lib/anagrams.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 def anagrams?(str1, str2)
-end
\ No newline at end of file
+if str1.sum == str2.sum then
+       puts 'True'
```
Comment (sumeetjain) [inline lib/anagrams.rb]: `puts "True"` is a lot like `puts "Banana"`: It's just displaying a string. There are two things wrong with this:
1. The goal of the method is not to _display_ text. It's to _return_ a value. So `return "True"` would be closer to correct.
2. You shouldn't be returning a _string_ at all. `"True"` is like `"Banana"` in that they're both just sets of characters. But remove the quotes and lowercase the 'T' and you get `true` (no quotes). _That_ is not a string at all. It's a **boolean**.

Comment (mfmeier) [inline lib/anagrams.rb]: I understand and made that change.

**Comment by sumeetjain** [comment]
Nice! This assignment is good to go. I'm still not sure what the problem with running `rake` in Cloud9 was. Let me know if that's still a problem with the next assignment.

**Pull request closed without merging**